### ISSUE TITLE
Overflowing Hour to Meridian Logic is Broken

### ISSUE THREAD

**Issue by paulstatezny** [zippyui/react-time-picker#6]
Hi there, thanks for making this React component!

I ran into a pretty glaring bug. Let me explain it with an example user interaction:

Say the component says `11:58 AM` and you are clicking the "up minute" button. It changes as such:

```
11:58 AM
11:59 AM
12:00 AM
00:01 PM
00:02 PM
...
```

Click back down, and it reads:

```
00:01 PM
00:00 PM
11:59 AM
11:58 AM
```

1 minute higher than `11:59 AM` is `12:00 PM`, and the reverse is true as well. There's no such thing as `00:MM PM` -- it should be `12:MM PM`.

You can use a JavaScript library like [moment.js](https://github.com/moment/moment/) to abstract these details away and fix it.

**Comment by zippyui-react** [comment]
Hi Paul and thanks for spotting that out. We're fixing that as soon as we can.
We've used moment.js for our [react-date-picker](https://zippyui.github.io/react-date-picker/), but for the time picker, the parsing logic is not very complicated, so we chose to do it ourselves and not add a rather big dependency.

**Comment by tptee** [comment]
Any news on this one?

**Comment by zippyui-react** [comment]
Will release a fix tomorrow, bumping the major version, since will have some/one API change(s), and also relying on momentjs for time parsing rather than use a built-in solution

**Comment by zippyui-react** [comment]
Hey, we've just released v 1.1.0 with a fix for this. No documented API changes.
@paulstatezny we integrated ntegrated momentjs after all.

the `onChange` prop has a moment object passed in as second argument (first remains a timeString)
